tests/baselines/reference/shorthandPropertyAssignmentInES6Module.errors.txt

```diff
@@ -0,0 +1,23 @@
+tests/cases/compiler/test.ts(2,19): error TS2307: Cannot find module './foo2'.
+tests/cases/compiler/test.ts(6,1): error TS2304: Cannot find name 'console'.
+tests/cases/compiler/test.ts(7,1): error TS2304: Cannot find name 'console'.

```

Comment:
Are these three errors part of the test? From reading the bug, it looks like only `foo1` and `const test = { x };` are needed and the rest can be removed.